.codecov.yml

```diff
@@ -14,3 +14,11 @@ coverage:
 
 ignore:
   - "test"
+  - "src/zimcheck.cpp"
```

Comment:
I don’t really get why you remove src code from codecov.

Reply:
Because zimcheck (as well zimdump, zimdiff) are files that provide a main() function. They will never be testable from the outside. Thats why IMO they should not count for codecov

Reply:
This is a bit much of assumption IMO because:
* many of these files have a lot of code (not only the main()
* you seems to consider that all automated tests a re unit tests, which is wrong

Reply:
I agree here. Even if they have a main function which make this difficult testable in this situation, this code *should* be tested. Ignore the files from codecov just hide this.

Reply:
Agree with you guys